charts/kof-child/templates/child-multi-cluster-service.yaml

```
@@ -90,34 +101,91 @@ spec:
       {{- end }}
 
       - name: kof-operators
-        namespace: {{ .Release.Namespace }}
+        namespace: {{ $.Release.Namespace }}
         template: kof-operators-{{ $version }}
         templateChain: kof-operators-{{ $version }}
         values: |
           {{`{{ $operatorsValuesFromAnnotation := index .Cluster.metadata.annotations "k0rdent.mirantis.com/kof-operators-values" | default "{}" | fromYaml }}`}}
-          {{`{{`}} $operatorsValuesFromHelm := `{{ .Values.operators | toYaml | nindent 10 }}` | fromYaml {{`}}`}}
+          {{`{{`}} $operatorsValuesFromHelm := `{{ $.Values.operators | toYaml | nindent 10 }}` | fromYaml {{`}}`}}
           {{`{{`}} $globalValuesFromHelm := `{{ $globalValues | toYaml | nindent 10 }}` | fromYaml {{`}}`}}
           {{`{{ mergeOverwrite (dict) $globalValuesFromHelm $operatorsValuesFromHelm $operatorsValuesFromAnnotation | toYaml | nindent 4 }}`}}
 
       - name: kof-collectors
-        namespace: {{ .Release.Namespace }}
+        namespace: {{ $.Release.Namespace }}
         template: kof-collectors-{{ $version }}
         templateChain: kof-collectors-{{ $version }}
         values: |
+        {{- if $istio }}
+          {{`{{ $regionalClusterName := getField "ChildConfig" "data.regional_cluster_name" }}`}}
```

Please update **all** `:=` assignments of all vars in `if`-s like this:

```suggestion
        {{ $regionalClusterName := "" }}
        {{- if $istio }}
          {{`{{ $regionalClusterName = getField "ChildConfig" "data.regional_cluster_name" }}`}}
```